content/lessons/B1_rusty_graphs/petgraph_sample_dfs.rs

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
https://github.com/mimuw-jnp2-rust/mimuw-jnp2-rust.github.io/blob/main/Cargo.toml#L88=

musisz tutaj dodać binarkę, bo widać że nie było odpalone cargo fmt na tym kodzie, a jak dodasz tam to się odpali automatycznie przy ./check.sh

Reply:
chociaż tak widzę teraz że to jest fragment kodu który ewidentnie się nie skompiluje, bo nie ma importów